packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -84,6 +86,13 @@ class CreateCommand extends FlutterCommand {
       },
       defaultsTo: null,
     );
+    argParser.addOption(
+      'sample',
+      abbr: 's',
+      help: 'Specifies the Flutter code sample to use as the main.dart for an application. Implies '
+        '--template=app.',
+      defaultsTo: null,
```

Comment:
`valueHelp: 'sample-id'`

(we should start consistently using `valueHelp` where appropriate, and back-fill existing args when we touch them.)